### ISSUE TITLE
Google search box takes focus without modifier key

### ISSUE THREAD

**Issue by shaunpatel** [televator-apps/vimari#38]
I noticed that the google search box continually grabs focus on the search results page. So if you are using vimari without a modifier key, any key press gets translated into the search box instead of as a vimari command. Any ideas on why this is occurring? 

I realize that adding a modifier key circumvents this issue, though I find it much easier to use vimari without the modifier.

**Comment by xudshen** [comment]
Turning off `Instant Results` works for me.

**Comment by gitfoxi** [comment]
Turning off Instant Results did not work for me.

**Comment by betoharres** [comment]
just select the radio option 
* ``Never show Instant results.``

but the real problem is when you cmd + Tab to make a new search in the address bar and the focus of the browser keeps in the address bar :rage: and I have to press Tab 4 times every time I do this so I can get focus in the page. 

I do this every 5 min so you can imagine how  irritating this is. I'm considering moving back to cvim in chrome

**Comment by gitfoxi** [comment]
@betoharres My mistake. That does fix my problem. I probably forgot to click `Save` in the Google Settings page before.

One good turn deserves another and I have a solution for your problem.

1. Goto Safari Preferences -> General
2. New tabs open with: Homepage
3. Homepage: www.google.com

Only drawback with this is that if you want to use something like Omnibar to assign quick shortcuts to different search engines, now you have to hit cmd-L first.

**Comment by betoharres** [comment]
@gitfoxi yeah thanks, I still would like to keep the blank new tab with my most visited websites but your solution solves my problem, thanks :+1:

**Comment by horak** [comment]
This worked for me as well! Not sure if this is common enough of a problem but if you guys like, I could put a section into the readme?

**Comment by danielcompton** [comment]
Is this still an issue? I can't reproduce this with the latest version.

**Comment by TribuneX** [comment]
I see the same issue. Pressing e.g. the `f` on the Google search result page does not show the shortcuts but instead inserts f into the search bar (since the focus is still on the Safari search bar).

**Comment by gth7754125** [comment]
> Is this still an issue? I can't reproduce this with the latest version.

It still a big problem.
